packages/react-scripts/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@fs/react-scripts",
-  "version": "8.8.4",
+  "version": "8.8.5",
```

This is a new feature, so let's minor bump
```suggestion
  "version": "8.6.0",
```

---

@joeycozza, was that supposed to have been a bump to "8.9.0"?